llama/patches/0007-add-mllama-support.patch

```diff
@@ -1,41 +1,57 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: jmorganca <jmorganca@gmail.com>
-Date: Thu, 17 Oct 2024 15:18:22 -0700
+Date: Tue, 8 Apr 2025 19:27:12 -0700
 Subject: [PATCH] add mllama support
 
-mllama adds cross-attention layers to the standard llama architecture
-it also requires a way to input a new tensor: cross_attention_state
-once per generation
-
-cross-attention layers don't change and so they are cached in the
-kv cache once per run
-
-remaining is to implement the cross attention mask
```

Comment:
It looks like this PR removes the commit messages for the patches. It's very important that we retain them, otherwise we'll have no idea if the original problem still applies in the future.